packages/p2p/src/components/orders/order-details/order-details.scss

```diff
@@ -42,11 +47,17 @@
             font-weight: bold;
             line-height: 1.5;
 
-            &--expired {
-                color: var(--text-less-prominent);
+            &--info {
+                color: #{$color-grey-1};
+            }
+            &--success {
+                color: #{$color-green-1};
+            }
+            &--danger {
+                color: #{$color-red-1};
             }
```

Comment:
Please use CSS variables. You may find these in `deriv-app/packages/shared/src/styles/themes.scss`

Reply:
@aaron-binary 
Done!